pkgs/by-name/ri/river/package.nix

```diff
@@ -15,14 +15,14 @@
 , wayland-protocols
 , wlroots_0_17
 , xwayland
-, zig_0_12
+, zig_0_13
```

Comment:
those overrides should be done in all-packages,nix

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults

since this PR didn't introduce this, this is optional

Reply:
What about the packages in `by-name`? They are not imported in `all-packages`.

Genuinely curious.

Reply:
follow the link I've provided 

specifically this

> So instead it is preferable to use the same generic parameter name libbar and override its value in [pkgs/top-level/all-packages.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix):
> 
> {
>   libfoo = callPackage ../by-name/so/some-package/package.nix {
>     libbar = libbar_2;
>   };
> }

normally by-name packages are not listed in all-packages.nix, but for such overrides they can be added to all-packages.nix